Formula/bento4.rb

```diff
@@ -0,0 +1,28 @@
+class Bento4 < Formula
+  desc "Full-featured MP4 format and MPEG DASH library and tools"
+  homepage "https://www.bento4.com/"
+  url "https://github.com/axiomatic-systems/Bento4/archive/v1.5.0-615.tar.gz"
+  version "1.5.0-615"
+  sha256 "109d48b75e7ba34d5a0cb98346c098d9e0d29d58b6bc27b90014dbc558a491a4"
+
+  conflicts_with "gpac", :because => "both install `mp42ts` binaries"
+
+  def install
+    binaries = %w(aac2mp4 avcinfo hevcinfo mp42aac mp42hevc mp42hls mp42ts
+    mp4compact mp4dcfpackager mp4decrypt mp4dump mp4edit mp4encrypt mp4extract
+    mp4fragment mp4info mp4mux mp4rtphintinfo mp4split mp4tag)
+
+    Dir.chdir "Build/Targets/any-gnu-gcc"
+    system "make", "AP4_BUILD_CONFIG=Release"
+
+    Dir.chdir "Release"
```

Comment:
Ah, never mind, it obviously had to be inside the `chdir … do` block. Now it would look like this:

    chdir "Build/Targets/any-gnu-gcc" do
      system "make", "AP4_BUILD_CONFIG=Release"
      binaries.each do |binary_name|
        bin.install "Release/#{binary_name}"
      end
    end

Is that acceptable?